io/net/virt-net/network_virtualization.py

```diff
@@ -465,11 +465,14 @@ def test_rmdev_viosfailover(self):
             self.validate_vios_command(
                 'rmdev -l %s' % vnic_backing_device, 'Defined')
 
-        after = self.get_active_device_logport(self.slot_num[0])
-        self.log.debug("Active backing device after: %s", after)
+        time.sleep(10)
```

Comment:
@vaishnavibhat is this still un-avoidable sleep here?
we can add wait.wait_for() if possible to avoid static sleep time.
I could see other static sleep timings too. So if possible we can replace them with wait.wait_for().